src/js/state/Search/selectors.js

```diff
@@ -15,7 +15,8 @@ export default {
       program: SearchBar.getSearchBar(state).previous,
       pins: SearchBar.getSearchBar(state).pinned,
       spanArgs: Tab.getSpanArgs(state),
-      space: Tab.spaceName(state)
+      spaceName: Tab.spaceName(state),
+      spaceID: Tab.spaceID(state)
```

Comment:
I know I'm to blame for straying from the convention (trying to make javascript like ruby), but are you willing to prefix any new selectors we make with "get". So here we'd have `Tab.getSpaceID()`?

Reply:
Ah yes, I prefer that pattern as well. Will adjust